frontend/src/component/project/Project/Project.tsx

```diff
@@ -186,33 +186,41 @@ export const Project = () => {
                             </PermissionIconButton>
                         </StyledDiv>
                     </StyledTopRow>
-                    <StyledColumn>
-                        <StyledProjectTitle>
-                            <div>
-                                <ConditionallyRender
-                                    condition={Boolean(project.description)}
-                                    show={
+                    <ConditionallyRender
+                        condition={!uiConfig?.flags?.newProjectOverview}
+                        // TODO: !!! Remove entire block when removing feature flag!
+                        show={
+                            <StyledColumn>
+                                <StyledProjectTitle>
+                                    <div>
+                                        <ConditionallyRender
+                                            condition={Boolean(
+                                                project.description
+                                            )}
+                                            show={
+                                                <StyledDiv>
+                                                    <StyledTitle data-loading>
+                                                        Description:&nbsp;
+                                                    </StyledTitle>
+                                                    <StyledText data-loading>
+                                                        {project.description}
+                                                    </StyledText>
+                                                </StyledDiv>
+                                            }
+                                        />
                                         <StyledDiv>
                                             <StyledTitle data-loading>
-                                                Description:&nbsp;
+                                                projectId:&nbsp;
```

Comment:
i've seen we use {' '} in other file in this PR

Reply:
+1